backends/advanced/src/advanced_omi_backend/utils/audio_chunk_utils.py

```
@@ -495,6 +495,108 @@ async def reconstruct_audio_segments(
+    Example:
+        >>> # Get first 60 seconds of audio
+        >>> wav_bytes = await reconstruct_audio_segment(conv_id, 0.0, 60.0)
+        >>> # Save to file
+        >>> with open("segment.wav", "wb") as f:
+        ...     f.write(wav_bytes)
+    """
+    from advanced_omi_backend.models.conversation import Conversation
+
+    # Validate inputs
+    if start_time < 0:
+        raise ValueError(f"start_time must be >= 0, got {start_time}")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)